assets/app/features/upgrade/controllers/upgrade-upgrade-administration-server.controller.js

```diff
@@ -24,6 +24,14 @@
             checkAdminUpgrade: checkAdminUpgrade
         };
 
+        activate();
+
+        function activate() {
+            // On page load we need to check to see if the upgrade is already running to set
+            // the button status and the update check running
+            checkAdminUpgrade();
```

Comment:
We need to test this functionality, somehow. I'm still not sure about the proper approach for this that we should follow here (and in the following steps). But is not part of the scope of the current card we've assigned this sprint.
It's part of Restore last step one, as detailed in https://trello.com/c/5fXGm1a7/45-2-27-restore-last-step (Has not been properly refined, estimated, prioritised nor committed yet. ;) )